src/vunnel/utils/http.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import time
+from typing import Any
+
+import requests
+
+DEFAULT_TIMEOUT = 30
+
+
+def get(
+    url: str,
+    logger: logging.Logger,
+    retries: int = 5,
+    backoff_in_seconds: int = 3,
+    timeout: int = DEFAULT_TIMEOUT,
+    **kwargs: Any,
+) -> requests.Response:
+    logger.debug(f"attempting get request on {url}")
```

Comment:
nit/suggestion: since this will show up a lot, the smaller the better
```suggestion
    logger.debug(f"http GET {url}")
```